### PR TITLE
Remove legacy Twitter support

### PR DESCRIPTION
**Pull request by gaeulbyul**
This PR does 2 things:
* Removes class WebTwitter which is no longer used in current Twitter.
* Don't change the Origin HTTP Header. It was intended to fix the API request problem on old Twitter. but it is also no longer needed on current Twitter.

(Sorry for the messy PR. I accidentally reverted my commits 😢  )

### CODE REVIEW

File: js/blockchain.js [hunk 1]
```diff
@@ -297,15 +89,12 @@ class MobileTwitter {
...
         }
-        let position = $(".GridTimeline-items").data('min-position');
-        let lastRequestTime = Date.now();
         let cursor = null;
         const _getIDData = () => {
             if (!finderRunning) return false;
             const count = 5000;
             let url = `${requestType}/ids.json?screen_name=${profileUsername}&count=${count}&stringify_ids=true`
             if (cursor) url += `&cursor=${cursor}`
-            lastRequestTime = Date.now();
```
Comment (mcclure) [inline js/blockchain.js:308]: Why did you remove these lines (300, 301, 308)?

**Comment by mcclure** [comment]
Thanks! :)
Two notes
- I am CC0ing (public domain) my additions to the plugin, can you either confirm you are submitting your changes as CC0 or state a different preference [such as being covered by Twitter Blockchain's original MIT license]
- If you want to be added to the CONTRIBUTORS file please add that change to your PR.

This patch also might be worth submitting to https://github.com/ceceradio/twitter-block-chain tho I think the author has not accepted any PRs in a little while.

File: js/blockchain.js (hunk 1 above)
Comment (gaeulbyul) [reply js/blockchain.js:308]: Because both `lastRequestTime` and `position` are not used anywhere, and `$(".GridTimeline-items")` element is not available in current Twitter.

**Comment by gaeulbyul** [comment]
> can you either confirm you are submitting your changes as CC0

Yes, It’s fine.

> If you want to be added to the CONTRIBUTORS file please add that change to your PR.

~~I’m at work right now so I’ll add it later.~~ done.

**Comment by mcclure** [comment]
Gonna merge this the next time I cut a release.
